packages/block-library/src/section/editor.scss

```diff
@@ -0,0 +1,55 @@
+$block-name: "section";
```

Comment:
Minor: this doesn't seem really useful to me, but I guess it's personal preference

Reply:
Guard against the number of times this block has been renamed 👍 But yes I take your point, it adds confusion.

Reply:
Resolved as of https://github.com/WordPress/gutenberg/pull/13964/commits/291fc8a06bf819221aacf90c9d31482a3d7e00cc